### ISSUE TITLE
Camera values being reset to default

### ISSUE THREAD

**Issue by jmp0** [gentlepumpkin/bi-aidetection#43]
This happens when running AI Tool as a service.

1. With AI Tool already started as a service, double-click aitool.exe to bring up the GUI.
2. Select a camera, change its Cooldown value, and click Save.
3. Close the GUI or just go to a different page.
4. Wait a few minutes.
5. Start the GUI back up and/or go back to the camera settings. 
6 Notice that Cooldown will be 2 minutes regardless of what value you entered before.

If you  stop the service and follow the above steps, or stop the service and manually modify the [cameraname].txt file, the settings will be preserved correctly.

**Comment by VorlonCD** [comment]
Correct.  Until we get a true service set up, the service and the GUI dont talk to each other so you do in fact have to stop the service before changing settings in GUI.

**Comment by cydia2020** [comment]
I had the same problem, my solution is to make the camera config file read only to the user that the service is running from.
